docker/hadoop/cdh/Dockerfile

```diff
@@ -89,11 +89,9 @@ RUN chmod 700 /usr/sbin/startup.sh && \
     chown -R hive:hive /opt/hive-scripts && \
     chmod +x /usr/sbin/install_hive.sh && \
     chmod +x /usr/sbin/install_ldap.sh && \
-    chmod +x /usr/sbin/mysql_configure.sh && \
     sync && \
     /usr/sbin/install_hive.sh && \
-    /usr/sbin/mysql_configure.sh && \
-    ln -sf /usr/share/java/mysql-connector-java.jar /usr/lib/hive/lib/mysql-connector-java.jar
+    ln -sf /usr/share/java/postgresql-jdbc4.jar /usr/lib/hive/lib/postgresql-jdbc4.jar
```

Comment:
There's a similar linking in `docker/hadoop/hdp/scripts/sbin/install_hive.sh`. I would like if they will be handled in one place, but that's probably out of scope of this PR.

Reply:
yes, the difference is that HIVE_HOME is not needed here